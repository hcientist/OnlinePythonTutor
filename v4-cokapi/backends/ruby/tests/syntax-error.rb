x = 1
y = 2
z = 3x
q = x + y
