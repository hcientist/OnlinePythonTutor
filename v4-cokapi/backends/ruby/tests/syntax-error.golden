{
  "code": "x = 1\ny = 2\nz = 3x\nq = x + y\n",
  "trace": [
    {
      "event": "uncaught_exception",
      "line": 3,
      "exception_msg": "syntax error, unexpected tIDENTIFIER, expecting end-of-input"
    }
  ]
}
